guzzle_php81_shim.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace GuzzleHttp;
+
+if (!function_exists('\GuzzleHttp\http_build_query')) {
```

Comment:
This seems problematic on drupal 9 since the order is backwards when composer-compile-plugin comes along, i.e. it gets defined here, and then guzzle itself tries to redefine it but of course it doesn't have this guard.

Reply:
I might be wrong about the reason, but there's some kind of redeclaration issue.